gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -13,9 +13,18 @@
             <div class="input-group">
               <input ng-if="editingDeviceNew()" name="deviceID" id="deviceID" class="form-control text-monospace" type="text" ng-model="currentDevice.deviceID" required="" valid-deviceid list="discovery-list" aria-required="true" />
               <div ng-if="!editingDeviceNew()" class="well well-sm form-control text-monospace" style="height: auto;" select-on-click>{{currentDevice.deviceID}}</div>
-              <div class="input-group-btn">
-                <button type="button" class="btn btn-default" data-toggle="modal" data-target="#idqr" ng-disabled="editingDeviceNew() && !deviceEditor.deviceID.$valid">
-                  <span class="fas fa-qrcode"></span>&nbsp;<span translate>Show QR</span>
+              <div id="shareDeviceIdButtons" class="input-group-btn">
+                <button type="button" class="btn btn-default" ng-click="copyToClipboard(currentDevice.deviceID)" ng-disabled="editingDeviceNew() && !deviceEditor.deviceID.$valid" tooltip data-original-title="{{ 'Copy' | translate }}">
```

Comment:
Somehow the "Copied!" tooltip re-appears for me when closing and re-opening this modal.  Maybe it should be cleared when closing the dialog?

Reply:
I can't reproduce this one. Can you describe the steps in detail? I've tested in Chromium and Firefox, but to no avail. I did find out that the script seems broken in Palemoon (which likely means that it doesn't work in older versions of Firefox either), but that's a different problem.

Reply:
No idea, I can't reproduce it anymore now.